hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceConfigurationBuilder.java

```diff
@@ -1377,6 +1382,33 @@ private static String getObjectName(ModelNode modelNode, OperationContext contex
         }
     }
 
+    private static List<OperationParam> getOpParamListFromOpNode(ModelNode modelNode) throws OperationFailedException {
+
+        List<OperationParam> ret = new ArrayList<>();
+
+        ModelNode params = modelNode.get("param");
```

Comment:
this is hardcoding "param" - need to use the Attribute class and resolve expressions. See "getString" usages as examples